lib/secure_cookies.rb

```diff
@@ -11,18 +9,15 @@ def initialize(app)
   def call(env)
     status, headers, body = @app.call(env)
 
-    if headers['Set-Cookie'].present?
-      cookies = headers['Set-Cookie'].split(COOKIE_SEPARATOR)
+    if (cookie_header = headers['Set-Cookie']).present?
+      cookies = cookie_header.split(COOKIE_SEPARATOR)
 
       cookies.each do |cookie|
         next if cookie.blank?
-        attributes = SECURE_COOKIE_ATTRIBUTES.zip(SECURE_COOKIE_REGEXES).reject do |_attr, regex|
-          cookie.match?(regex)
-        end.map(&:first)
 
-        next if attributes.empty?
-
-        cookie << attributes.join
+        cookie << '; Secure' if env['HTTPS'] == 'on' && !cookie.match?(/; Secure/i)
+        cookie << '; HttpOnly' if !cookie.match?(/; HttpOnly/i)
+        cookie << '; SameSite=Lax' if !cookie.match?(/; SameSite/i)
```

Comment:
I went with just direct strings and skipped the constant array since we needed different behavior for `Secure`

Also now `SameSite` will check for any SameSite value and not just lax